src/axom/core/examples/core_acceleration.cpp

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) 2017-2021, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+/*! \file core_containers.cpp
+ *  \brief This example code is a demonstration of the Axom Core containers.
+ */
+
+/* This example code contains snippets used in the Core Sphinx documentation.
+  * They begin and end with comments such as
+  *
+  * timer_start
+  * timer_end
+  *
+  * each prepended with an underscore.
+  */
+
+// Axom includes
+#include "axom/core/StackArray.hpp"
+#include "axom/core/execution/execution_space.hpp"
+#include "axom/core/execution/for_all.hpp"
+#include "axom/core/execution/synchronize.hpp"
+#include "axom/core/memory_management.hpp"
+
+#ifdef WIN32
+  #include "windows.h"
+void sleep(int numSeconds)
+{
+  int numMilliSecs = numSeconds * 1000;
+  Sleep(numMilliSecs);
+}
+#else
+  #include <unistd.h>  // for sleep()
+#endif
+
+// C/C++ includes
+#include <iostream>
+#include <vector>
+
+void demoMemoryManageBasic()
+{
+  // _membasic_start
+  int *dynamic_memory_array;
+  int *dyn_array_dst;
+  int len = 20;
+
+  //Allocation looks similar to use of malloc() in C -- just template
+  //return type instead of casting.
+  dynamic_memory_array = axom::allocate<int>(len);
+
+  for(int i = 0; i < len; i++)
+  {
+    dynamic_memory_array[i] = i;
+  }
+
+  for(int i = 0; i < len; i++)
+  {
+    std::cout << i << " Current value: " << dynamic_memory_array[i] << std::endl;
+  }
+
+  //Now, a copy operation.
```

Comment:
this operation is allocation, not copy.

Reply:
I think this made sense to me as a matter of story-telling, because the 'copy section' of the example had been entered, but it definitely doesn't represent the line immediately following. I can see the issue now, and will handle it.

Reply:
Comment was removed.